pkg/machinery/config/encoder/documentation.go

```diff
@@ -74,10 +75,18 @@ func (d *Doc) Describe(field string, short bool) string {
 
 // Example represents one example snippet for a type.
 type Example struct {
+	sync.Once
```

Comment:
do we ever copy/pass `Example` by value? (vs. by reference)

Reply:
we _shouldn't_ but let me check for sure.

Reply:
That should be checked by vet's copylock check

Reply:
yeah, should be good.